Resources/Prototypes/Maps/Pools/deathmatch.yml

```
@@ -1,4 +1,5 @@
-- type: gameMapPool
-  id: DeathMatchMapPool
-  maps:
-  - MeteorArena
+- type: gameMapPool
```

yamllint

[error] wrong new line character: expected \n

(new-lines)